proto/src/determined/job/v1/job.proto

```diff
@@ -129,3 +129,14 @@ message QueueStats {
   // Number of scheduled jobs in the queue.
   int32 scheduled_count = 2;
 }
+
+// Aggregate statistics for a queue.
+message AggregateQueueStats {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "period_start", "seconds" ] }
+  };
+  // Number of queued jobs in the queue.
```

Comment:
Just checking on the PR. In this section, update the comments on these fields and rebuild the TypeScript bindings.

Reply:
Thank you for catching that, fixed